lib/fizzbuzz.rb

```diff
@@ -1,11 +1,17 @@
 class SuperFizzBuzz
 
   def run(input)
-
-    #Implement your code here
-
+  	if input % 15 == 0
+  		"FizzBuzz"
+  	elsif input % 5 == 0
+  		"Buzz"
+  	else input % 3 == 0
+  		"Fizz"
+  	end
   end
 
+# How can I refactor this and does it actually bear refactoring? What's the advantage of this approach vs. hash access and vice versa?
```

Comment:
Personally, I would write a method named `divisible?` that checks if a number is divisible by another, and then use it in the `run` method. I think indexing a hash is an interesting approach, but this is already a pretty good solution